docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,521 @@
+    "- Generate label quality scores for each example in the dataset. \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Quickstart \n",
+    "\n",
+    "Cleanlab uses two inputs to generate scores for labels in the dataset:\n",
+    "- `labels`: NumPy array of given labels in the dataset. labels[i] should contain label for `i`-th example. \n",
+    "- `predictions`: NumPy array of predictions generated through your favorite regressor. predictions[i] should contain predicted value for `i`-th example. \n",
+    "\n",
+    "If you already have predictions from your regressor, you can generate label quality scores for each example using the code below: \n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">\n",
+    "\n",
+    "```python \n",
+    "\n",
+    "from cleanlab.regression.rank import get_label_quality_scores\n",
+    "label_quality_scores = get_label_quality_scores(labels, predictions)\n",
+    "\n",
+    "```\n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install dependencies and import them \n",
+    "You can use `pip` to install all the packages required for this tutorial as follows:\n",
+    "\n",
+    "`!pip install cleanlab xgboost`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# Package versions we used: xgboost==1.7.2\n",
+    "\n",
+    "dependencies = [\"cleanlab\", \"xgboost\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = \" \".join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from xgboost import XGBRegressor\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "from cleanlab.regression.rank import get_label_quality_scores\n",
+    "\n",
+    "np.set_printoptions(suppress=True)"
```

Comment:
move this to a separate hidden cell, also want to set the numpy seed in this same hidden cell as that may affect sklearn cross-val-predict?

The hidden cell can just have comment at the top:

# This cell is hidden on docs.cleanlab.ai